File: ecomonitoring/src/main/java/com/ecomonitoring/src/Person.java
/**
 * Базовий клас для всіх дійових осіб системи
 */
public abstract class Person {
    protected String name;
    protected String role;

    public Person(String name, String role) {
        this.name = name;
        this.role = role;
    }

    public String getName() {
        return name;
    }

    public String getRole() {
        return role;
    }

    public void printInfo() {
        System.out.println("Особа: " + name + ", Роль: " + role);
    }
}

File: src/Person.py
# Базовий клас для всіх дійових осіб системи
class Person:
    def __init__(self, name: str, role: str):
        self.name = name
        self.role = role

    def get_name(self) -> str:
        return self.name

    def get_role(self) -> str:
        return self.role

    def print_info(self):
        print(f"Особа: {self.name}, Роль: {self.role}")
